packages-exp/auth-compat-exp/src/recaptcha_verifier.ts

```diff
@@ -32,7 +34,7 @@ export class RecaptchaVerifier
     app: FirebaseApp = firebase.app()
   ) {
     // API key is required for web client RPC calls.
-    impl.assertFn(app.options?.apiKey, impl.AuthErrorCode.INVALID_API_KEY, {
+    _assert(app.options?.apiKey, impl.AuthErrorCode.INVALID_API_KEY, {
```

Comment:
shouldn't the signature be updated here?

Reply:
We don't have an auth object yet here